.github/workflows/developers_onboarding_notification.yml

```diff
@@ -0,0 +1,116 @@
+name: Celebrating Contributions
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  comment_on_merged_pull_request:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set Environment Variables
+        env:
+          AUTHOR: ${{ github.event.pull_request.user.login }}
+          REPO: ${{ github.event.repository.name }}
+          OWNER: ${{ github.event.repository.owner.login }}
+        run: |
+          echo "AUTHOR=${AUTHOR}" >> $GITHUB_ENV
+          echo "REPO=${REPO}" >> $GITHUB_ENV
+          echo "OWNER=${OWNER}" >> $GITHUB_ENV
+
+      - name: Count Merged Pull Requests
+        id: count_merged_pull_requests
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            try {
+              const author = process.env.AUTHOR;
+              const repo = process.env.REPO;
+              const owner = process.env.OWNER;
+              const { data } = await github.rest.search.issuesAndPullRequests({
+                q: `repo:${owner}/${repo} type:pr state:closed author:${author}`
+              });
+              const prCount = data.items.filter(pr => pr.pull_request.merged_at).length;
+              core.exportVariable('PR_COUNT', prCount);
+            } catch (error) {
+              core.setFailed(`Error counting merged pull requests: ${error.message}`);
+            }
+
+      - name: Comment on the Merged Pull Request
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            try {
+              const prCount = parseInt(process.env.PR_COUNT);
+              const author = process.env.AUTHOR;
+              const mention = 'amir-naveh'; 
+              const prNumber = context.payload.pull_request.number;
+              const repo = process.env.REPO;
+
+              function getRandomEmoji() {
+                const emojis = ['🎉', '🚀', '💪', '🌟', '🏆', '🎊', '🔥', '👏', '🌈', '🚂'];
+                return emojis[Math.floor(Math.random() * emojis.length)];
+              }
+
+              function getMessage(count) {
+                const emoji = getRandomEmoji();
+                switch(count) {
+                  case 1:
+                    return `${emoji} **Fantastic work @${author}!** Your very first PR to ${repo} has been merged! 🎉🥳\n\n` +
+                           `You've just taken your first step into open-source, and we couldn't be happier to have you onboard. 🙌\n` +
+                           `If you're feeling adventurous, why not dive into another issue and keep contributing? The community would love to see more from you! 🚀\n\n` +
+                           `For any support, feel free to reach out to the developer onboarding lead: @${mention}. Happy coding! 👩‍💻👨‍💻`;
```

Comment:
`${emoji} **Fantastic work @${author}!** Your very first PR to ${repo} has been merged! 🎉🥳\n\n` +
                           `If you're feeling adventurous, why not dive into another issue and keep contributing? The Classiq community would love to see more from you! 🚀\n\n` +
                     Happy coding! 👩‍💻👨‍💻`;